exercises/rail-fence-cipher/rail_fence_cipher_test.rb

```diff
@@ -0,0 +1,94 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'rail_fence_cipher'
+
+# rubocop:enable all
+class RailFenceCipherTest < Minitest::Test
+  def test_encode_with_empty_string
+    assert_equal '', RailFenceCipher.encode('', 4)
```

Comment:
Interestingly, this doesn't only give people the opportunity to use a module, but it does free up the solution to be pretty open.